src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,125 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+import { getData } from '../../Api';
+
+export const FindMovie = ({ onMovies }) => {
+  const [value, setValue] = useState('');
+  const [errorFind, setError] = useState(false);
+  const [previewMovie, setMovie] = useState(null);
```

Comment:
```suggestion
  const [error, setError] = useState(false);
  const [previewMovie, setPreviewMovie] = useState(null);
```
The names of the functions that set the values in `useState` should start with `set` and continue with the name of the value